pkg/crc/config/config.go

```diff
@@ -200,6 +211,7 @@ func (c *Config) Get(key string) SettingValue {
 		}
 	case string:
 		value = cast.ToString(value)
+	case Path:
```

Comment:
Imo this should be `value = Path(cast.ToString(value))`

Reply:
Doing this fails the existing test case, had to also add a stringer method for the Path type